include/JSObjectProxy.hh

```diff
@@ -0,0 +1,192 @@
+/**
+ * @file JSObjectProxy.hh
+ * @author Caleb Aikens (caleb@distributive.network)
+ * @brief JSObjectProxy is a custom C-implemented python type that derives from dict. It acts as a proxy for JSObjects from Spidermonkey, and behaves like a dict would.
+ * @version 0.1
+ * @date 2023-05-03
+ *
+ * Copyright (c) 2023 Distributive Corp.
+ *
+ */
+
+#include <jsapi.h>
+
+#include <Python.h>
```

Comment:
```suggestion
#include <Python.h>

#include <unordered_map>
```

required for macOS build

https://github.com/Distributive-Network/PythonMonkey/actions/runs/5340441182/jobs/9680265397#step:7:452